flink-kubernetes/src/main/java/org/apache/flink/kubernetes/utils/KubernetesUtils.java

```diff
@@ -86,6 +93,44 @@
 
     private static final YAMLMapper yamlMapper = new YAMLMapper();
```

Comment:
After all other comments are addressed, I believe we do not need so many inessential changes in `KubernetesUtils.java`.